### ISSUE TITLE
[Accessibility] Microsite - Color Contrast fails to meet WCAG conformance

### ISSUE THREAD

**Issue by emmaindal** [backstage/backstage#21885]
### 📜 Description

Example: Selection drop-downs, “are you using Backstage”, privacy info, links in docs text

Users with low vision may find controls inoperable or be unable to read on-page text

Some of these elements returned contrasts of 1.27:1 (such as the privacy paragraph) thus new color pallets may need to be chosen by design

### 👍 Expected behavior

Color contrast succeeds. One can use [Contrast Checker](https://webaim.org/resources/contrastchecker/) to verify this. 

### 👎 Actual Behavior with Screenshots

(couldn't find an example of the issue of when the audit was first conducted but here is another one)

Example of links:
<img width="514" alt="Screenshot 2023-12-15 at 14 02 43" src="https://github.com/backstage/backstage/assets/25153114/206bb1f7-7e17-4a5e-bac9-5647ddfe1f77">

### 👟 Reproduction steps

1. Go to https://backstage.io/
2. [Run the Lighthouse browser extension ](https://chromewebstore.google.com/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk?pli=1)

Alternatively:
1. Run the microsite locally
2. [Run the Lighthouse CLI locally when developing new features](https://backstage.io/docs/accessibility/#run-the-lighthouse-cli-locally-when-developing-new-features)

### 👀 Have you spent some time to check if this bug has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/backstage/backstage/blob/master/CODE_OF_CONDUCT.md)

### Are you willing to submit PR?

None

**Comment by npiyush97** [comment]
@emmaindal @vinzscam suggestion changing #36baa2 link color to #15473E work with both #e3e3e3 and #ffffff

**Comment by npiyush97** [comment]
adding underline is enough to solve this.
